src/main/java/org/jabref/logic/importer/fileformat/ACMPortalParser.java

```diff
@@ -1,19 +1,5 @@
 package org.jabref.logic.importer.fileformat;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.net.CookieHandler;
-import java.net.CookieManager;
-import java.net.MalformedURLException;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
-
```

Comment:
For some strange reason IntelliJ automatically moves the imports to the wrong place. Looks like a bug in IntelliJ.

Reply:
Can you test this behaviour? If you move these imports to the right place?

Reply:
I also saw that - with IntelliJ 2025. With 2024, it works fine. Which version are you using?

Reply:
I just updated a few days ago.

Reply:
Maybe I can somehow downgrade. I'm sorry but I won't be avail until Sunday (or only occasionally)

Reply:
Used my laptop with an older version.

Reply:
Bugfix https://youtrack.jetbrains.com/articles/IDEA-A-2100662392/IntelliJ-IDEA-2024.3.4.1-243.25659.59-build-Release-Notes?utm_source=product&utm_medium=link&utm_campaign=TBA